lib/src/base-styles/_nx-btn.scss

```diff
@@ -6,48 +6,67 @@
  */
 @use '../scss-shared/nx-container-helpers';
 @use '../scss-shared/nx-text-helpers';
+@use '../scss-shared/nx-dark-mode-helpers';
 
-%button-focus-outline {
-  outline-offset: -5px;
-  outline: 2px solid;
+$transparent-button-outline-offset: calc(0px - var(--nx-border-width-button) - 4px);
+
+@mixin solid-btn-styling($variant) {
+  background-color: var(--nx-color-#{$variant}-button-background);
+  border-color: transparent;
+  color: var(--nx-color-#{$variant}-button-text);
+
+  &:hover {
+    background-color: var(--nx-color-#{$variant}-button-background-hover);
+  }
+
+  &:focus:not(:active) {
+    outline-color: var(--nx-color-#{$variant}-button-focus-outline);
+    outline-offset: -5px;
```

Comment:
Shouldn't this be `-4px`? Also could it be a computation based on some variables? The amount by which the focus borders are supposed to be inset is always `2px` right? So maybe that should be a CSS variable as well, and this `outline-offset` could be calculated from that variable and `--nx-focus-outline-width`

Reply:
> Shouldn't this be `-4px`? Also could it be a computation based on some variables? The amount by which the focus borders are supposed to be inset is always `2px` right? So maybe that should be a CSS variable as well, and this `outline-offset` could be calculated from that variable and `--nx-focus-outline-width`

This had been partially resolved previously ([deac080](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/deac0808835df98a9e5aada5b9e11842e073737b)) but created the css variable for the 2px inset here as well: [913d7f1](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/913d7f1ffccd46447a83a70c425b8bbd1cabc07e)